More2Life.xcodeproj/project.pbxproj

```diff
@@ -313,6 +315,8 @@
 		E27BDA821F54C54500112A4C /* HalfModalTransitioningDelegate.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = HalfModalTransitioningDelegate.swift; sourceTree = "<group>"; };
 		E27BDA881F54C66900112A4C /* BuyModalViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = BuyModalViewController.swift; sourceTree = "<group>"; };
 		E27BDA8A1F56098A00112A4C /* MainButton.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = MainButton.swift; path = ../More2Life/MainButton.swift; sourceTree = "<group>"; };
+		E2E709E41F9FFE6700AF6A2C /* DonationFeedItem+CoreDataClass.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = "DonationFeedItem+CoreDataClass.swift"; path = "/Users/bkingsford/Developer/m2l-iOS/Shared/DonationFeedItem+CoreDataClass.swift"; sourceTree = "<absolute>"; };
```

Comment:
`sourceTree = "<absolute>"` seems wrong. I wonder why your Xcode is marking it absolute. It should be on `<group>`